packages/google_sign_in/google_sign_in_platform_interface/lib/src/types.dart

```diff
@@ -363,6 +363,29 @@ class AuthenticationResults {
   final AuthenticationTokenData authenticationTokens;
 }
 
+/// Parameters for the clearAuthorizationTokens method.
+@immutable
+class ClearAuthorizationTokensParams {
```

Comment:
Question: do we always need to introduce an additional type for parameters? Is it specifically for pigeon?

Reply:
Would it be possible to mark the class `final` (so no need for the runtimeType check in `==`), or using an extension type?